django_ipam/base/admin.py

```diff
@@ -47,7 +50,25 @@ class Media:
         css = {'all': ('django-ipam/css/admin.css',)}
 
 
+class IpAddressAdminForm(forms.ModelForm):
+    def __init__(self, *args, **kwargs):
+        super(IpAddressAdminForm, self).__init__(*args, **kwargs)
+        self.fields["subnet"].help_text = _('Select a subnet and the first available IP address '
+                                            'will be automatically suggested in the ip address field')
+
+
 class AbstractIpAddressAdmin(TimeReadonlyAdminMixin, ModelAdmin):
+    form = IpAddressAdminForm
+    change_form_template = "admin/django-ipam/ip_address/change_form.html"
+
+    def add_view(self, request, form_url='', extra_context=None):
+        extra_context = {'get_first_available_ip_url': reverse('ipam:get_first_available_ip', args=('0000',))}
+        return super(AbstractIpAddressAdmin, self).add_view(request, form_url, extra_context)
+
+    def change_view(self, request, object_id, form_url="", extra_context=None):
+        extra_context = {'get_first_available_ip_url': reverse('ipam:get_first_available_ip', args=('0000',))}
```

Comment:
this line is identically exact as line 65, please avoid duplication, create a method or an attribute that stores this info.